client/src/components/QAhelperFunctions.js

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable no-var */
+const _ = require('underscore');
+//combines the text of the Q and its A's together, then filters them based on the search bar entry
+
+const reviewFilter = function (Qs, filter) {
```

Comment:
this function is responsible for taking the search bar entry and filtering out any Questions that do not have the term in either itself or any of the answers